src/Stepper/Stepper.jsx

```diff
@@ -0,0 +1,304 @@
+import React, {PropTypes} from 'react';
+
+import {getMuiTheme} from '../styles';
+import Paper from '../paper';
+
+const Stepper = React.createClass({
+  propTypes: {
+    /**
+     * The current active step index which passed by parent component.
+     */
+    activeStepIndex: PropTypes.number,
+    /**
+     * Children should be Step type.
+     */
+    children: PropTypes.node,
+    /**
+     * The icon will show when the step is completed.
+     */
+    completedIcon: PropTypes.node,
+    /*
+     * Override inline-style of the content container.
+     */
+    containerStyle: PropTypes.object,
+    /**
+     * Create suitable icon for step base on state of the step.
+     */
+    createIcon: PropTypes.func,
+    /**
+     * If true, it will be horizontal stepper.
+     */
+    horizontal: PropTypes.bool,
+    /**
+     * Callback function that is fired when the header of step is touched.
+     */
+    onHeaderStepTouch: PropTypes.func,
+    stepHeadersWrapperStyle: PropTypes.object,
+    /**
+     * Override the inline-styles of the root element.
+     */
+    style: PropTypes.object,
+    /**
+     * Callback function that is fired when re-render to update the background of left avatar.
+     If not passed, it will use default theme
+     */
+    updateAvatarBackgroundColor: PropTypes.func,
```

Comment:
Hmm, I think the user may want to custom more state of the stepper 
Success, Active or more ( Warning when the validation fails,  Hovered, Disabled) etc...
So i added this function to update the background color